src/doc/functional/_enrolment.rst

```diff
@@ -2,4 +2,197 @@
+    :noindex:
+
+    Retrieve the attributes/status of a document validation.
+
+    **Authorization**: :todo:`To be defined`
+
+    :param str issuanceID: The ID of the issuance.
+    :param dict options: The processing options. Supported options are transactionID.
+    :return: a status indicating success or error, and in case of success the document validation status.
+
+.. py:function:: createBiometricValidationRequest(biometricID, biometricData, callback, options)
+    :noindex:
+
+    Request validation of biometric data.
+
+    **Authorization**: :todo:`To be defined`
+
+    :param str documentID: The ID of the biometric data.
+    :param documentData: The content and attributes of the biometric to be validated.
+    :param uri callback: The address of a service to be called when the result is available.
+    :param dict options: The processing options. Supported options are transactionID.
+    :return: a status indicating success or error.  In the case of success, a biometric validation identifier.
+
+.. py:function:: getBiometricValidationRequest(biometricValidationID, options)
+    :noindex:
+
+    Retrieve the attributes/status of a biometric validation.
+
+    **Authorization**: :todo:`To be defined`
+
+    :param str issuanceID: The ID of the issuance.
+    :param dict options: The processing options. Supported options are transactionID.
+    :return: a status indicating success or error, and in case of success the biometric validation status.
+
+.. py:function:: createBiographicValidationRequest(biographicID, biometricData, callback, options)
+    :noindex:
+
+    Request validation of biographic data.
+
+    **Authorization**: :todo:`To be defined`
+
+    :param str documentID: The ID of the biographic data.
+    :param documentData: The content and attributes of the biographic to be validated.
+    :param uri callback: The address of a service to be called when the result is available.
+    :param dict options: The processing options. Supported options are transactionID.
+    :return: a status indicating success or error.  In the case of success, a biographic validation identifier.
+
+.. py:function:: getBiographicValidationRequest(biographicValidationID, options)
+    :noindex:
+
+    Retrieve the attributes/status of a biographic validation.
+
+    **Authorization**: :todo:`To be defined`
+
+    :param str issuanceID: The ID of the issuance.
+    :param dict options: The processing options. Supported options are transactionID.
+    :return: a status indicating success or error, and in case of success the biographic validation status.
+
+Options
+"""""""
+
+.. list-table:: Population Registry Services Options
+    :header-rows: 1
+    :widths: 25 75
+
+    * - Name
+      - Description
+
+    * - ``transactionID``
+      - A string provided by the client application to identity the request being submitted.
+        It is optional in most cases. When provided, it can be used for tracing and debugging.
+
+Data Model
```

Comment:
I think we need a little bit of details about the issuance data (result of `getCredentialIssuanceRequest`) and the validation results. For example, the validation of a portrait could include information such as "eyes closed".